README.md

```diff
@@ -57,11 +57,16 @@ This sample demonstrates how to use BCPii HWIREST in a TSO/E REXX or ISV REXX in
 
 <br/>[**Example-Energy-REXX**](https://github.com/IBM/zOS-BCPii/tree/master/Example-Energy-REXX)
 
-This samples uses BCPii HWIREST REXX interface to retrieve energy management information for a target CPC. The results are stored in .csv format, in a member in a z/OS data set.
+This sample uses BCPii HWIREST REXX interface to retrieve energy management information for a target CPC. The results are stored in .csv format, in a member in a z/OS data set.
 
 <br/>[**Example-Audit-REXX**](https://github.com/IBM/zOS-BCPii/tree/master/Example-Audit-REXX)
 
-This samples uses BCPii HWIREST REXX interface to audit LPARs on a target CPC. The results are stored in .csv format, in a member in a z/OS data set.
+This sample uses BCPii HWIREST REXX interface to audit LPARs on a target CPC. The results are stored in .csv format, in a member in a z/OS data set.
+
+<br/>[**Example-Crypto-REXX**](https://github.com/IBM/zOS-BCPii/tree/master/Example-Crypto-REXX)
+
+This sample uses BCPii HWIREST REXX interface to retrieve crypto information from image activation profiles for a target CPC. The results are stored in .csv format, in a member in a z/OS data set. Note the crypto properties are valid on z16 processors or higher.
```

Comment:
This sample uses BCPii HWIREST REXX interface to retrieve crypto information from image activation profiles associated with LPARs located on a specific CPC.

Reply:
Done